src/site/styles/layout.less

```diff
@@ -93,11 +93,46 @@ input {
     position: relative;
 }
 
-.action-confirmation {
-    position: absolute;
-    top: 0;
-    left: 0;
-    background: white;
+.action-confirmation { 
```

Comment:
Sure, I understand your concern, but I'm not sure I agree with your conclusion. We can discuss this another time however.